src/sheet/SheetEditor.js

```diff
@@ -350,25 +350,10 @@ export default class SheetEditor extends AbstractEditor {
     let editorSession = this.getEditorSession()
     let cell = this._getAnchorCell()
     let newValue = this._formulaEditorContext.node.getText()
-    let { anchorRow, anchorCol} = this._getSheetSelection()
-
-    // Select cell below
-    anchorRow += 1
-
     editorSession.transaction(tx => {
       tx.set(cell.getPath(), newValue)
-      tx.setSelection({
-        type: 'custom',
-        customType: 'sheet',
-        data: {
-          type: 'range',
-          anchorRow,
-          anchorCol,
-          focusRow: anchorRow,
-          focusCol: anchorCol
-        },
-        surfaceId: this.refs.sheet.getId()
-      })
+      let newSel = this.refs.sheet.shiftSelection(1, 0, false)
+      tx.setSelection(newSel)
```

Comment:
@oliver---- better like that?

Reply:
Yeah. Better that we don't see the internal selection data.
And with re viewport: SheetComponent should be smart enough to update the viewport automatically
If it doesn't we will fix it later.